### PR TITLE
[6.x] Core/Spell: scripted spell 114698 (Summon Amberleaf Troublemaker)

### PR DESCRIPTION
**Pull request by Lopfest**
**Changes proposed**:
- changed MaxAffectedTargets of spell 114710 to 1
- scripted summon location of spell 114698 

**Target branch(es)**: 6x

**Tests performed**: tested in-game with 2 players simultaneously

**Referenced PR**: #16811 

**Spell description**:
NPC [Alysa Cludsinger](http://www.wowhead.com/npc=59642/aysa-cloudsinger) periodically casts spell 114710 on players in range. These players then cast summon spell 114698 on a location defined by spell script. (no invisible triggers are present at the spawning location)

### CODE REVIEW

File: src/server/scripts/Pandaria/zone_the_wandering_isle.cpp [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            else
+                radius = 7;
+            float angle = M_PI * (urand(0, 7) / 7.f);
+            float x = 1181.75f + radius * sin(angle);
+            float y = 3444.5f + radius * cos(angle);
+            float z = 102.9385f;
+
+            GetHitDest()->Relocate(x, y, z);
+
+            Position const SpawnPosition = { x, y, z, 3.285759f };
```
Comment (joschiwald) [inline src/server/scripts/Pandaria/zone_the_wandering_isle.cpp]: check out spell target destination select hook

**Comment by Aokromes** [comment]
@Lopfest  read https://github.com/TrinityCore/TrinityCore/pull/16989#discussion_r60829725

**Comment by Lopfest** [comment]
@Aokromes yes, I already tried that. It is not possible to do it that way. Spell doesn't hit any target.

**Comment by Lopfest** [comment]
already contained in PR #17021

**Pull request closed without merging**